aspnetcore/security/authentication/accconfirm.md

```diff
@@ -14,7 +14,7 @@ uid: security/authentication/accconfirm
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Joe Audette](https://twitter.com/joeaudette)
 
-This tutorial shows you how to build an ASP.NET Core app with email confirmation and password reset. This tutorial is **not** a beginning topic. You should be familiar with:
+This tutorial shows you how to build an ASP.NET Core app with email confirmation and password reset. This tutorial isn't a beginning topic. You should be familiar with:
```

Comment:
I prefer the original in this (and rare cases). Too many customers complained the tutorial was difficult to follow. It's inherently difficult.